koboldai_settings.py

```diff
@@ -1234,7 +1234,13 @@ def __init__(self, socketio, koboldai_var):
         self.userscripts = []     # List of userscripts to load
         self.last_userscripts = []  # List of previous userscript filenames from the previous time userscripts were send via usstatitems
         self.corescript  = "default.lua"  # Filename of corescript to load
-        self.gpu_device  = 0      # Which PyTorch device to use when using pure GPU generation
+        if 'IPEX' in os.environ:
```

Comment:
given that both else statements do the same thing it might be cleaner to do
```py
if "IPEX" in os.environ and os.environ["IPEX"]:
```
or, even better,
```py
if os.environ.get("IPEX"):
```
(since `.get()` returns None if the key isnt found)

Reply:
I tought that it will complain.